package.json

```diff
@@ -27,10 +44,9 @@
     "test:cover:CI": "npm run test:CI -- --coverage && cat ./coverage/lcov.info | node_modules/coveralls/bin/coveralls.js",
     "test:watch": "jest --watchAll",
     "open:cover": "npm run test:cover && opn ./coverage/lcov-report/index.html",
-    "analyze-bundle": "babel-node ./tools/analyzeBundle.js"
+    "analyze-bundle": "babel-node ./tools/analyzeBundle.js",
+    "rebuildNodeSass": "npm rebuild node-sass"
```

Comment:
I don't think this npm command adds much. Where you have it documented, I would probably just write run `npm rebuild node-sass`.